docs/react/week-2/lesson.md

```diff
@@ -259,9 +259,17 @@ Finally, let's look at how we get hold of the `count` variable:
 const [count, setCount] = useState(0);
 ```
 
-To fully understand this bit of code, we first have to understand _destructuring_. Let's look at [this blog post by Wes Bos about array destructuring](https://github.com/wesbos/es6-articles/blob/master/19%20-%20Destructing%20Arrays.md).
+If you remember back to [JS Core 3 Week 3](https://syllabus.codeyourfuture.io/js-core-3/week-3/lesson#destructuring-syntax), we covered _destructuring_. As a reminder, you can assign variables to parts of an array with destructuring:
```

Comment:
Could this be tweaked to be a relative link?

Reply:
👍 Done in https://github.com/CodeYourFuture/syllabus/pull/136/commits/b364f37d48f5b1919a5228e09a950a61689dd2d5